apps/web/vibes/soul/primitives/accordion.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+
+import * as AccordionsPrimitive from '@radix-ui/react-accordion'
+
+export const Root = AccordionsPrimitive.Root
+
+type Props = React.ComponentPropsWithoutRef<typeof AccordionsPrimitive.Item> & {
+  /**
+   * The title of the accordion item. It will be added to the accordion's header.
+   */
+  title: string
+
+  children: React.ReactNode
+}
+
+export function Item({ title, children, ...rest }: Props) {
```

Comment:
Should we include a `displayName`?

Reply:
Won't the display name be `Item` since this is a function declaration? I've only seen explicit `displayName` necessary when using `forwardRef` with an anonymous function resulting in an unnamed component

Reply:
Ah yes, that makes sense.